tests/containers/podman_integration.pm

```diff
@@ -91,22 +106,20 @@ sub run {
     # user / remote
     #
 
-    unless (is_sle_micro) {
-        assert_script_run "cp -r test/system.orig test/system";
-        @skip_tests = split(/\s+/, get_required_var('PODMAN_BATS_SKIP') . " " . get_var('PODMAN_BATS_SKIP_USER_REMOTE', ''));
-        foreach my $test (@skip_tests) {
-            script_run "rm test/system/$test.bats";
-        }
-
-        $log_file = "bats-user-remote.tap";
-        assert_script_run "echo $log_file .. > $log_file";
-        background_script_run "podman system service --timeout=0";
-        script_run "env PODMAN=/usr/bin/podman QUADLET=$quadlet hack/bats --rootless --remote | tee -a $log_file", 2600;
-        parse_extra_log(TAP => $log_file);
-        assert_script_run "rm -rf test/system";
-        script_run 'kill %1';
+    assert_script_run "cp -r test/system.orig test/system";
+    @skip_tests = split(/\s+/, get_required_var('PODMAN_BATS_SKIP') . " " . get_var('PODMAN_BATS_SKIP_USER_REMOTE', ''));
+    foreach my $test (@skip_tests) {
+        script_run "rm test/system/$test.bats";
     }
 
+    $log_file = "bats-user-remote.tap";
+    assert_script_run "echo $log_file .. > $log_file";
+    background_script_run "podman system service --timeout=0";
+    script_run "env PODMAN=/usr/bin/podman QUADLET=$quadlet hack/bats --rootless --remote | tee -a $log_file", 2600;
+    parse_extra_log(TAP => $log_file);
+    assert_script_run "rm -rf test/system";
+    script_run 'kill %1';
```

Comment:
Why `kill` and not `wait`?

Reply:
`wait` would hang forever waiting for `podman system service --timeout=0` ran as background above.